mkdocs.yml

```diff
@@ -28,6 +28,7 @@ nav:
       - Actions:
           - documentation/actions/index.md
           - AMQP: documentation/actions/amqp.md
+          - Kafka: documentation/actions/kafka.md
```

Comment:
It's better to add the entry after Mongo to keep alphabetical order,
just keep "non-action" entries (like "extension" or "other") at the end.